app/src/main/java/nodomain/freeyourgadget/gadgetbridge/util/BengaliLanguageUtils.java

```diff
@@ -157,7 +159,13 @@ public static String transliterate(String txt) {
 
         Matcher m = bengaliRegex.matcher(txt);
         StringBuffer sb = new StringBuffer();
+        String lastChar = "";
+        Boolean lastHadComposition = false;
+        Boolean lastHadKaar = false;
+        Boolean nextNeedsO = false;
```

Comment:
All these and below should be lower-case `boolean`s.

Reply:
Fixed this. All set to merge. Can you review and merge please?